lua/relative-toggle/init.lua

```diff
@@ -15,11 +15,15 @@ local function create_autocmd(event, opts)
 end
 
 ---@param relative boolean #whether relativenumber should be set
-local function set_relativenumber(relative)
+---@param redraw boolean #whether to redraw the screen
+local function set_relativenumber(relative, redraw)
     local in_insert_mode = vim.api.nvim_get_mode().mode == "i"
 
     if vim.o.number then
         vim.opt.relativenumber = relative and not in_insert_mode
+        if redraw then
+            vim.cmd("redraw")
```

Comment:
I couldn't find a lua API equivalent for this 😞

Reply:
It's okay. Currently, there is no lua API for `redraw` command

```suggestion
            vim.cmd "redraw"
```